hbase-server/src/main/java/org/apache/hadoop/hbase/replication/SystemTableWALEntryFilter.java

```diff
@@ -17,16 +17,24 @@
  */
 package org.apache.hadoop.hbase.replication;
 
+import org.apache.hadoop.hbase.security.access.PermissionStorage;
+import org.apache.hadoop.hbase.security.visibility.VisibilityConstants;
 import org.apache.hadoop.hbase.wal.WAL.Entry;
 import org.apache.yetus.audience.InterfaceAudience;
 
 /**
- * Skips WAL edits for all System tables including hbase:meta.
+ * Skips WAL edits for all System tables including hbase:meta except hbase:acl.
  */
 @InterfaceAudience.Private
 public class SystemTableWALEntryFilter implements WALEntryFilter {
   @Override
   public Entry filter(Entry entry) {
+    if (
+      entry.getKey().getTableName().equals(PermissionStorage.ACL_TABLE_NAME)
```

Comment:
It is a bit confusing to users that why only hbase:acl can be replicated but other system tables are not? Please provide more information here, like how do you plan to use this feature?

Thanks.

Reply:
Hi Duo, Thanks for the review. 

> why only hbase:acl can be replicated but other system tables are not
   As of now, only 2 tables can be allowed for replication - hbase:acl and hbase:labels
        1. hbase:meta - not to be replicated
        2. hbase:canary - for current cluster
        3. hbase:namespace - Deprecated and moved to meta
        4. hbase:quota - related to namespace, quota for the current cluster usage
        5. hbase:rsgroup - contains hostnames

> how do you plan to use this feature
    We are trying to set up an active-active deployment and we want this hbase:acl table to be replicated so that any connected client to either one cluster will have the same access control information.

Thanks.

Reply:
Please add this to the javadoc of this class?

Thanks.

Reply:
@Kota-SH Please see the comment above? Thanks.